### ISSUE TITLE
Class probability is high but object score is very low

### ISSUE THREAD

**Issue by tsindhuja** [pjreddie/darknet#2048]
While training on Yolov3 tiny, I'm getting the following during training after 2000 iterations on 200 images on a single class
`v3 (mse loss, Normalizer: (iou: 0.75, cls: 1.00) Region 16 Avg (IOU: 0.645552, GIOU: 0.632020), Class: 0.987351, Obj: 0.037266, No Obj: 0.000512, .5R: 0.818182, .75R: 0.363636, count: 11, class_loss = 1.304511, iou_loss = 0.956496, total_loss = 2.261008    
v3 (mse loss, Normalizer: (iou: 0.75, cls: 1.00) Region 23 Avg (IOU: 0.622210, GIOU: 0.589366), Class: 0.981931, Obj: 0.045332, No Obj: 0.000417, .5R: 0.710526, .75R: 0.210526, count: 38, class_loss = 4.476244, iou_loss = 4.540621, total_loss = 9.016865 
`
